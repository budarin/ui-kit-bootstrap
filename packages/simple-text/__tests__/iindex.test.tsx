import React from 'react';
import renderer from 'react-test-renderer';
import SimpleText from '../src/index';

test('0 should be 0', () => {
    const tree = renderer.create(<SimpleText>'Text'</SimpleText>).toJSON();

    expect(tree).toMatchSnapshot();

    expect(0).toBe(0);
});
