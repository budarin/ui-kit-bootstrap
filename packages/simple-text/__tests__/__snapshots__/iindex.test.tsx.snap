// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`0 should be 0 1`] = `
<span>
  'Text'
</span>
`;
